gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.3.1-all.zip
+distributionUrl=https://nexus3.ucds.io/repository/thirdparty/org/gradle/distributions/gradle/5.5.1/gradle-5.5.1-bin.zip
```

Comment:
I think we should be using the open source `services.gradle` endpoint since this is a github.com repo?